src/wallet/KintoWallet.sol

```diff
@@ -45,9 +46,9 @@ contract KintoWallet is Initializable, BaseAccount, TokenCallbackHandler, IKinto
     address[] public override owners;
     address public override recoverer;
     mapping(address => bool) public override funderWhitelist;
-    mapping(address => mapping(address => uint256)) private _tokenApprovals;
     mapping(address => address) public override appSigner;
     mapping(address => bool) public override appWhitelist;
+    IKintoApp public override appRegistry = IKintoApp(address(0));
```

Comment:
I'm a bit confused by this... When would this be set?

Reply:
changed and made immutable. Will be set in the new impl deployment